packages/clay-icon/src/ClayIcon.js

```diff
@@ -26,15 +26,6 @@ ClayIcon.STATE = {
 	 */
 	elementClasses: Config.string(),
 
-	/**
-	 * Flag to indicate if the svg is focusable or not
-	 * @default false
-	 * @instance
-	 * @memberof ClayIcon
-	 * @type {?bool}
-	 */
-	focusable: Config.bool().value(false),
```

Comment:
hey @carloslancha, @jbalsas instead of removing this here, we can trigger a warning when someone uses it, stating that it is being depreciated. What do you think? And we can remove it completely in the next release...

Reply:
Hey @matuzalemsteles, that would be fine by me. Could you take this one and apply that change? The problem will be that we don't have an easy way to flag that from the template itself, so we will only be able to show the warning when used from js...

Reply:
Sure